### DISCUSSION TITLE
How can I simply ignore an auto-generate file in a directory managed by `.chezmoiexternal`?

### DISCUSSION THREAD

**Discussion by kitos9112** [twpayne/chezmoi#3632] [Q&A] [answered]
Hello 👋🏼!

## Problem Statement
I leverage `.chezmoiexternal` to keep track of multiple remote Git repos and other tarballs. I download pre-compiled `fonts` from a Git repo as a release tarball that then I untar in the right local directory. Once these fonts get scanned by my local Linux system, `fc-cache` places a tiny single-line checksum in this file (.uiid) which I cannot do anything about (unless getting rid of it with a script before run `chezmoi apply/update`).

I'm almost sure this got answered in a couple of older issues or discussions already... however, I cannot find the right one or even discern whether there is a current solution or workaround to this.

Probably https://github.com/twpayne/chezmoi/discussions/3493? I just need to ignore the whole content.

Many thanks in advance!

**Comment by twpayne** [comment, marked as answer]
You should be able to add the files to your `.chezmoiignore` file, something like:

```
my/path/to/fonts/*.uiid
```

**Reply by kitos9112** [reply]
Alright! I had to add another `.chezmoiignore` to my dotfiles repo within the actual "home" location.

I was oppositely using .chezmoiignore, basically to ignore files getting copied from my repo to my local home directory.
